g/gojsonschema/gojsonschema_rhel_8.3.sh

```diff
@@ -0,0 +1,39 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : gojsonschema
+# Version       : v1.2.0
+# Source repo   : https://github.com/xeipuuv/gojsonschema.git
+# Tested on     : RHEL 8.3
+# Language      : GO
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Raju.Sah@ibm.com
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ------------------------------------------------------------------------------
+set -e
+PACKAGE_NAME=gojsonschema
+PACKAGE_VERSION={1:-v1.2.0}
+PACKAGE_URL=https://github.com/xeipuuv/gojsonschema.git
+
+yum update -y && yum install -y git golang
```

Comment:
I don't think we need a "yum update here", any specific reason for including it?

Reply:
'yum update' can be removed , it will not affect the script.